docs/packages/omc3/workflow.md

```diff
@@ -1,7 +1,141 @@
 # OMC3 Software Workflow
 
-!!! todo
-    Description of a typical analysis workflow. With easy examples for first-timers.
+## The OMC3 Analysis Workflow
 
-!!! todo
-    Also add smaller examples for individual entry-points, scripts and plotting.
+A typical workflow with `omc3` consists in performing analysis of measurement or simulation files, and eventually calculating corrections to apply.
+While `omc3` provides python modules to handle the different aspects of these tasks, it also provides entrypoint scripts to be called from the commandline.
+
+The first step in the workflow consists in a harmonic frequency analysis performed with the `harpy` module, while the second one is an optics analysis performed with the `measure_optics` module.
+The table below shows a general analysis workflow from BPM turn-by-turn measurements or simulation file to results, as well as the corresponding files at each step:
+
+| Workflow & Files       |                        |                        |                        |                                   |
+| :--------------------- | :--------------------: | :--------------------: | :--------------------: | --------------------------------: |
+| Turn-by-Turn BPM Data  | :material-arrow-right: | Frequency Spectra      | :material-arrow-right: | Various Lattice Optics Parameters |
+| SDDS files:  `*.sdds`  | :material-arrow-right: | TFS files: `*.lin[xy]` | :material-arrow-right: | TFS files: `*.tfs`                |
+
+In this page, we will go through the essential steps in preparing and performing an analysis, by going from start to finish with a simple example.
+
+!!! example  ""
+    For our example, we will start from data obtained in `MAD-X` with the `TRACK` command for the LHC machine with a scenario adapted from the 2018 machine configurations.
+    Changes from the nominal scenario in your simulation could be including errors tables, orbit bumps, speculative magnet errors, additional elements etc.
+    It is also easy for the reader to follow along if starting from measurements files.
+
+## Preparing Data for Analysis
+
+Analysis in `omc3` is mostly done from measurement data, which has to be given in the LHC's [SDDS][sdds] binary format.
+If your starting point is one or more measurement files in `SDDS` format from a similar machine, then you are set.
+However, if your data is in another format you will need to first convert it to `SDDS`.
```

Comment:
nop, every reader from `tbt_converter` is also in `omc3`, so you can read it in straight away ..

Reply:
That's the feedback I'm after, I learn as I go :D

Is the `tbt_converter` for simulation files only like a `trackone` file then?

Reply:
you'd only need to do that if you want to analyse the data also with `beta-beat.src` .. in which case on either side some hacks are required

Reply:
> 
> Is the `tbt_converter` for simulation files only like a `trackone` file then?

kinda yes, with the converter you have also the option to add noise, which obv only makes sense when you have simulation data
you can convert basically every format to `.sdds` in case you have some analysis set up .. which would also be `Beta-Beat GUI`

Reply:
and converter also has the option to use average in case of multi-particle simulations (in case one wants to include decoherence in the tracking) .. but MAD-X now has the `only_average` option in tracking, so this one may be redundant

Reply:
Resolved with f65d02c